app/services/provider_router.py

```
@@ -0,0 +1,347 @@
+            field_mapping: Mapping of field names to translate
+            source_lang: Source language code
+            target_lang: Target language code
+            api_keys: Dictionary of provider API keys
+            context: Optional context for translation
+            
+        Returns:
+            Translated collection data
+        """
+        translated_data = collection_data.copy()
+        
+        for field_name, field_path in field_mapping.items():
```

Ruff (0.11.9)

Loop control variable `field_name` not used within loop body

Rename unused `field_name` to `_field_name`

(B007)